pmm-app/src/shared/components/hooks/parameters.hook.ts

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from 'react';
+import { getLocationSrv } from '@grafana/runtime';
+
+export const useUrlState = (key: string, defaultValue?: string): [string, (parameter: string) => void] => {
```

Comment:
This name is too generic. A more semantic name would be `useQueryParams`. 
@tiagomotasantos @nicolalamacchia Unless there is a better name :)